sos/plugins/block.py

```diff
@@ -32,6 +41,9 @@ def setup(self):
             "ls -lanR /sys/block"
         ])
 
+        if self.get_option("all"):
```

Comment:
For me I always like to try and get everything I could possibly want to help debug a problem in the first go around rather than going back to the customer to ask for more data.